### ISSUE TITLE
Implementation of qml virtual keyboard

### ISSUE THREAD

**Issue by PlayingKarrde** [mmatyas/pegasus-frontend#728]
Would it be worth implementing this: https://doc.qt.io/qt-5/qtvirtualkeyboard-index.html do you think? I was considering writing my own but seems like a decent amount of work when it already exists.

**Comment by iwubcode** [comment]
Is this a duplicate of #258 ?  I do think it'd be nice for controllers to have a way to search

**Comment by mmatyas** [comment]
Yes, there have been plans to add support for this, it's just that this module is somewhat different than the rest of the Qt Quick stuff, it's a pain to set it up and bundle correctly, and by default very touch-oriented. Will see if I can make this work.
